enterprise/tpa_pipeline.py

```diff
@@ -63,13 +64,13 @@ def handle_enterprise_logistration(backend, user, **kwargs):
     """
     LOGGER.info(f'Beginning enterprise logistration for LMS user {user.id}')
     request = backend.strategy.request
-    enterprise_customer = get_enterprise_customer_for_running_pipeline(
-        request,
-        {
-            'backend': backend.name,
-            'kwargs': kwargs
-        }
-    )
+    pipeline = {'backend': backend.name, 'kwargs': kwargs}
```

Comment:
this should work too I think it would be a slight bit cleaner if we can reuse the provider_id logic between this method and the get_enterprise_customer_for_running_pipeline method in case we change logic in the future, then we have to update in both places


ideally can we do sth like this?

```
def get_enterprise_customer_for_running_pipeline(request, pipeline):
    """
    Get the EnterpriseCustomer associated with a running pipeline.
    """
    sso_provider_id = resolve_provider_id(request, pipeline)
    return get_enterprise_customer_for_sso(sso_provider_id)
```

then in your code you can also just call

```
 sso_provider_id = resolve_provider_id(request, pipeline)
```